install/scripts/php-conf-fix.sh

```diff
@@ -22,6 +39,8 @@ then
     sed -i 's/^\(post_max_size\s*=\s*\).*$/\1701M/' php.ini
     echo 'Setting max upload filesize to 700M'
     sed -i 's/^\(upload_max_filesize\s*=\s*\).*$/\1700M/' php.ini
+    echo Setting timezone to $TIMEZONE
+    sed -i "s@.*date.timezone =.*@date.timezone = $TIMEZONE@" php.ini
```

Comment:
please use the same delimeters as above to be consistent. It might also be good to use single quotes to prevent globbing. I.e. use
```bash
    sed -i 's/.*date.timezone =.*/date.timezone = '"$TIMEZONE"/' php.ini
```